### ISSUE TITLE
PdfInfo.Create method lacks meaningful exception

### ISSUE THREAD

**Issue by tomasr78** [dlemstra/Magick.NET#843]
The `PdfInfo.Create` method can fail for several reasons but all failures return the same `MagickDelegateErrorException` exception and the exception message always the same:

`FailedToExecuteCommand `"C:\Projects\cara\gs\gswin64c.exe" -q -dQUIET -dSAFER -dBATCH -dNOPAUSE -dNOPROMPT --permit-file-read="C:/Users/Tomas/AppData/Local/Temp/CARA/nl3u38gZ/ZnY5cOmG.pdf" -sPDFPassword="qwert" -c "(C:/Users/Tomas/AppData/Local/Temp/CARA/nl3u38gZ/ZnY5cOmG.pdf) (r) file runpdfbegin pdfpagecount = quit"' (1) @ error/ghostscript-private.h/ExecuteGhostscriptCommand/74`

So basically it is not possible to know why the method is failed:

1. Password required for protected PDF.
2. The password incorrect.
3. The PDF is damaged

**Comment by dlemstra** [comment]
It might be possible to include the output of the excution in an exception message but that still means you will need to parse the message of the exception yourself.

**Comment by tomasr78** [comment]
That would be fine, I am already doing the MagickDelegateErrorException.Message parsing on PDF to Image to find out what exception is thrown.

**Comment by dlemstra** [comment]
In the next version of Magick.NET you will get the output of Ghostscript when the command fails. You will need to get that information from the exception(s) in the `RelatedExceptions`.
